src/context.rs

```diff
@@ -1,21 +1,22 @@
 use crate::ast::Value;
 use crate::schema::Schema;
-use fnv::FnvHashMap;
 use uuid::Uuid;
 
+type FastHashMap<K, V> = fnv::FnvHashMap<K, V>;
```

Comment:
This may not be the common pattern in Rust.

In Rust, something like `Pin<Box<dyn Future<Output = Result<T, Box<dyn Error + Send>>> + Send` might have an alias because of the crazy generic bounds.

Reply:
reverted.